### PR TITLE
Mvc ui inconsistencies with angular

### PR DESCRIPTION
**Pull request by iyilm4z**
Fixes [this](https://github.com/aspnetboilerplate/aspnetboilerplate/issues/5573).

### CODE REVIEW

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Roles/Index.js [hunk 1]
```diff
@@ -39,21 +39,16 @@
         columnDefs: [
             {
                 targets: 0,
-                className: 'control',
```
Comment (demirmusa) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Roles/Index.js:42]: Why did you remove that?

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Tenants/Index.js [hunk 2]
```diff
@@ -39,27 +39,22 @@
         columnDefs: [
             {
                 targets: 0,
-                className: 'control',
```
Comment (demirmusa) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Tenants/Index.js:42]: Why did you remove that?

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Components/SideBarMenu/UserMenuItemExtensions.cs [hunk 3]
```diff
@@ -9,8 +9,10 @@ public static class UserMenuItemExtensions
         public static IOrderedEnumerable<UserMenuItem> OrderByCustom(this IEnumerable<UserMenuItem> menuItems)
         {
             return menuItems
-                .OrderBy(menuItem => menuItem.Order)
-                .ThenBy(menuItem => menuItem.DisplayName);
+                .OrderBy(menuItem => menuItem.Order);
+            
+            // Uncomment below line to order menu items by DisplayName too
```
Comment (demirmusa) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Components/SideBarMenu/UserMenuItemExtensions.cs:14]: I think we can open it back.

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Users/Index.js [hunk 4]
```diff
@@ -39,32 +39,27 @@
         columnDefs: [
             {
                 targets: 0,
-                className: 'control',
```
Comment (demirmusa) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Users/Index.js:42]: Why did you remove that?

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Components/SideBarMenu/UserMenuItemExtensions.cs (hunk 3 above)
Comment (iyilm4z) [reply aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Views/Shared/Components/SideBarMenu/UserMenuItemExtensions.cs:14]: ok

**Comment by iyilm4z** [comment]
@demirmusa you are right, i'll fix it as soon as possible.
